corehq/apps/registration/templates/registration/mobile/mobile_worker_account_confirmation_success_sms.txt

```diff
@@ -0,0 +1,10 @@
+{% load i18n %}
+{% blocktrans %}Hello {{ name }},{% endblocktrans %}
```

Comment:
Instead of using `{% blocktrans %}` for each paragraph, maybe use one `{% blocktrans %}` for the whole message. It would give translators more context. For example, in some languages, a translator might choose to use a different word to end an email than "Thanks" but without knowing that this was the ending for an email, that context might be lost.